vms/platformvm/state/state.go

```diff
@@ -1302,7 +1297,11 @@ func (s *state) syncGenesis(genesisBlk block.Block, genesis *genesis.State) erro
 			return err
 		}
 
-		potentialReward := s.rewards.Calculate(
+		potentialReward := reward.Calculate(
+			s.cfg.RewardConfig.MaxConsumptionRate,
+			s.cfg.RewardConfig.MinConsumptionRate,
+			s.cfg.RewardConfig.MintingPeriod,
+			s.cfg.RewardConfig.SupplyCap,
```

Comment:
just pass s.cfg.RewardConfig here instead of exploding its attributes

Reply:
Responded above